elements/pfe-styles/src/pfe-context.scss

```diff
@@ -7,3 +7,13 @@
         --context: #{$context};
     }
 }
+
+@each $color in (darker, accent, complement, base, lighter, lightest) {
```

Comment:
Added the background color application here to add some more value to the context styles - I needed these in the baseline images to remove the pfe-band dependency and still be able to update the background color easily.